packages/@aws-cdk/aws-sagemaker/test/integ.endpoint-config.ts

```diff
@@ -0,0 +1,78 @@
+import * as path from 'path';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core';
+import { IntegTest } from '@aws-cdk/integ-tests';
+import * as sagemaker from '../lib';
+
+/*
+ * Stack verification steps:
+ * aws sagemaker describe-endpoint-config --endpoint-config-name <endpoint config name>
+ *
+ * The above command will result in the following output.
+ *
+ *   {
+ *     "EndpointConfigName": "EndpointConfig...",
+ *     "EndpointConfigArn": "arn:aws:sagemaker:...",
+ *     "ProductionVariants": [
+ *         {
+ *             "VariantName": "firstVariant",
+ *             "ModelName": "ModelWithArtifactAndVpcModel...",
+ *             "InitialInstanceCount": 1,
+ *             "InstanceType": "ml.m5.large",
+ *             "InitialVariantWeight": 1.0
```

Comment:
watch out! the decimal gets dropped in the template for all of these

Reply:
Please see my [note above](https://github.com/aws/aws-cdk/pull/22865#discussion_r1019678347).